admin_app/src/app/page.tsx

```diff
@@ -18,21 +19,24 @@ export default function Home() {
   const [cardToEdit, setCardToEdit] = useState<Content | null>(null);
   const [newCardText, setNewCardText] = useState("");
 
-  const deleteCard = (id: string) => {
-    fetch(`${backendUrl}/content/${id}/delete`, {
-      method: "DELETE",
-      headers: get_api_headers(),
-    }).then((response) => {
-      if (response.ok) {
-        const newCardList = cards.filter(
-          (card: Content) => card.content_id !== id,
-        );
-        setCards(newCardList);
-        setFilteredCards(newCardList);
-      } else {
-        throw new Error("Could not delete " + id);
-      }
-    });
+  const deleteCard = (e: React.MouseEvent, id: string) => {
+    e.stopPropagation();
+    if (window.confirm("Are you sure you want to delete the content?")) {
```

Comment:
Not a big fan of these kind of pop-ups. Can we make it a modal?

![image](https://github.com/IDinsight/aaq-core/assets/14125957/dc1c1bda-3edf-48a8-822a-2c2cdb11d9a0)

Reply:
Or can we a better name and icon for it?